index.html

```diff
@@ -11568,6 +11569,11 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>The <code>aria-haspopup</code> property is an enumerated type. <a>User agents</a> MUST treat any value of <code>aria-haspopup</code> that is not included in the list of allowed values, including an empty string, as if the value <code>false</code> had been provided. To provide backward compatibility with ARIA 1.0 content, user agents MUST treat an <code>aria-haspopup</code> value of <code>true</code> as equivalent to a value of <code>menu</code>.</p>
 				<p><a>Assistive technologies</a> SHOULD NOT expose the <code>aria-haspopup</code> property if it has a value of <code>false</code>.</p>
 				<p class="note">A <rref>tooltip</rref> is not considered to be a popup in this context.</p>
+				<p class="note"><code>aria-haspopup</code> is most relevant to use when there is a visual indicator in the element that triggers the popup. 
+					For example, most menu buttons are styled with a downward pointing triangle or chevron, which indicates to sighted users that the button will display a menu when clicked. 
```

Comment:
```suggestion
					For example, many controls styled with a downward pointing triangle or chevron, or represented by three consecutive dots, have become standard visual indicators that a popup will display when the control is activated. 
```

Reply:
I like your new wording @scottaohara - it gives many more examples with very few extra words. :)

Does it make sense to say "containing" three consecutive dots instead of "represented by" three consecutive dots? That would allow for such things as "Open..." instead of just the "..." of a horizontal or vertical kebab.

Reply:
yeh, good point @carmacleod 

i removed the word "represented", as i think think that allows for both interpretations (containing or represented by) to be inferred.

Reply:
So then does it become "styled by three consecutive dots"?